lzss/regress_test.go

```diff
@@ -27,7 +27,7 @@ var refValues = map[string]refValue{
 		lzssRatio: 3.55,
 	},
 	"./testdata/blobs/5-1128897": {
-		lzssRatio: 7.26,
+		lzssRatio: 7.17,
```

Comment:
interesting that this change actually make a worse compression ratio for this blob. 
It means if we considered the uncompressed input size after 1 << 20 (1mB), it was beneficial to have backrefs of len 20, instead of 21 now.

Reply:
I wouldn't worry about this one since due to "constraint constraints" 🤪 the highest effective compression ratio we can achieve is 6.23.

Reply:
no, with this change of 21 bits instead of 20, something we could compress to 5.3 could go to 5.1